src/lib/drivers/extension.js

```diff
@@ -145,7 +145,7 @@ class ExtensionDriver extends Driver {
     return new Promise((resolve, reject) => {
       chrome.tabs.query(currentTab, tabs => {
         if (chrome.runtime.lastError) {
-          return reject(chrome.runtime.lastError.message);
+          return reject(chrome.runtime.lastError);
```

Comment:
these two lines were changed in #296 and @paulirish mentioned another bug that happened to fix in https://github.com/GoogleChrome/lighthouse/pull/296#discussion_r62380771. Is this reverting and bringing back that bug?